bibliopixel/animation/sequence.py

```diff
@@ -47,4 +48,4 @@ def step(self, amt=1):
                 self.index = 0
 
         if not self.completed and self.animations:
-            self.current_animation.run_all_frames()
+            self.current_animation.start()
```

Comment:
Ah...nevermind. That commented line I removed was forcing them to be threaded. Not anymore.
BTW... start() wasn't in BP2. It was run()
Ok ok... the reason I changed it to start() was when I thought I had to force threaded operation. But then I fixed that by moving around the stop_event stuff. 
I changed it back to run_all_frames() now, which works fine.